schemas/data/measure.schema.json

```diff
@@ -53,5 +53,5 @@
     }
   ],
   "required": ["xdm:value"],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎 This seems to use @type and @id incorrectly